src/executor/graph_executor.cc

```diff
@@ -981,6 +986,41 @@ Graph GraphExecutor::InitGraph(nnvm::Symbol symbol,
   // setup gradient
   nnvm::Graph g = InitFullGraph(symbol, grad_req_types);
 
+#if MXNET_USE_CUDA && !defined(_WIN32)
+  if (dmlc::GetEnv("MXNET_USE_FUSION", true) && default_ctx.dev_mask() == Context::kGPU) {
```

Comment:
nit: the second test is faster than the relatively much more slow GetEnv() call, so maybe better to do the GetEnv() second?

Reply:
Sure, can do that.